client/src/components/FormInput.js

```diff
@@ -0,0 +1,53 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+import EmailIcon from '../assets/email.svg';
+import PasswordIcon from '../assets/password.svg';
+import InputField from './InputField';
+
+const Container = styled.div`
+  width: 22%;
+  height: 50px;
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  background-color: transparent;
+  border-width: 1px;
+  border-color: #961f56;
+  border-top-style: hidden;
+  border-right-style: hidden;
+  border-left-style: hidden;
+  border-bottom-style: solid;
```

Comment:
you could achieve the same effect with:
```suggestion
  border: 1px #961f56;
  border-style: hidden hidden solid hidden;
```